pygmo/core.cpp

```diff
@@ -692,7 +695,12 @@ BOOST_PYTHON_MODULE(core)
     inv.def(bp::init<unsigned, unsigned>((bp::arg("weeks") = 4u, bp::arg("sample_size") = 10u)));
     inv.def(
         bp::init<unsigned, unsigned, unsigned>((bp::arg("weeks") = 4u, bp::arg("sample_size") = 10u, bp::arg("seed"))));
-
+    // CEC 2013.
+    auto cec2013_ = pygmo::expose_problem<cec2013>(
+        "cec2013", "__init__(prob_id = 1, dim = 2, data_dir = 'input_data/')\n\nThe CEC 2013 problem suite.\n\n"
```

Comment:
I'd prefer if we had all the docstrings in the cpp file as raw strings, as it's much easier to edit them if needed.